package.json

```diff
@@ -65,6 +65,7 @@
     "husky": "^8.0.3",
     "lint-staged": "^13.2.2",
     "prettier": "^2.8.8",
+    "prettier-plugin-organize-imports": "^3.2.2",
```

Comment:
I didn't know this one, instead, I use [import/order](https://github.com/import-js/eslint-plugin-import/blob/main/docs/rules/order.md) which allows customizing the sorting.

I'm okay with either one ✌️

Reply:
I can install that one, the thing is, I don't really care about the order itself, I just want it to be sorted somehow and for unused imports to be removed automatically 😅

Reply:
So we are aligned! The order is useful for super large/custom projects, for this one, traditional sorting is perfect